parcels/kernels/TEOSseawaterdensity.py

```diff
@@ -0,0 +1,90 @@
+"""Collection of pre-built sea water density kernels"""
+import math
+
+
+__all__ = ['polyTEOS10_bsq']
```

Comment:
See comment above. Also Capitalise the function here (although this would break backwards-compatibility)?